pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -63,18 +64,29 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 				if isHealthyOperator(&item) {
 					continue
 				}
+				failingPods := []corev1.Pod{}
 				for _, namespace := range namespacesForOperator(&item) {
 					pods, err := i.coreClient.Pods(namespace).List(metav1.ListOptions{})
 					if err != nil {
 						klog.V(2).Infof("Unable to find pods in namespace %s for failing operator %s", namespace, item.Name)
+						continue
 					}
 					for i := range pods.Items {
 						if isHealthyPod(&pods.Items[i]) {
 							continue
 						}
+						failingPods = append(failingPods, pods.Items[i])
 						records = append(records, record.Record{Name: fmt.Sprintf("config/pod/%s/%s", pods.Items[i].Namespace, pods.Items[i].Name), Item: PodAnonymizer{&pods.Items[i]}})
 					}
 				}
+				networkEvents, err := handlePendingPodsNetworkEvents(failingPods, i.coreClient)
+				if err != nil {
+					klog.V(2).Infof("Unable to gather network events: %v", err)
+					continue
+				}
+				for i := range networkEvents {
+					records = append(records, record.Record{Name: fmt.Sprintf("config/events/%s/network", networkEvents[i].Namespace), Item: EventAnonymizer{&networkEvents[i]}})
```

Comment:
Prefer this just be events, not network related.  We'll want other types of events no matter what.